documentation/development/modularity-and-pbo-structure.md

```diff
@@ -10,23 +10,23 @@ order: 4
 
 Main principles:
 - As much stuff as possible should be modular
-- Strive to make as much stuff as possible run-time toogable. Adding/removing PBOS would still be requiring to toogle any feature relying on config changes.
+- Strive to make as much stuff as possible run-time togglable. Adding/removing PBOS would still be requiring to toggle any feature relying on config changes.
 
 ### PBO Structure
 
 Main principles:
 
 - Try to keep single module dependencies as much as possible
-- InteractionMenu would be the requirement for most modules.
-- Anything that is 100% config should require common and not interaction.
+- Interact_Menu would be the requirement for most modules.
```

Comment:
In fact this is outdated; should be "Interaction" instead of "interact_menu" on all ocurrences